draft-wood-cfrg-aead-limits.md

```diff
@@ -96,7 +105,14 @@ does describe limits on its inputs, but these are formulated as strict
 functional limits, such as the maximum length of inputs, which are determined by
 the properties of the underlying AEAD composition.  Degradation of the security
 of the AEAD as a single key is used multiple times is not given a thorough
-treatment.
+treatment. These limits might also be influenced by the number of "users" of
+a given key. In the traditional setting, there is one key shared between a two
+parties, and any limits on the maximum length of inputs or encryption operations
+apply to that single key. However, in practice, there are often many users with
+independent keys. In this "multi-user" setting, the attacker is assumed to have
+done some offline work to help break security (confidentiality or integrity)
+of a single key chosen at random. As a result, AEAD algorithm limits may depend
```

Comment:
I think that we want to break this out into a different paragraph at a minimum.

The way that I explain mu limits is to talk about "any" vs. "specific".  That is, if you care about the confidentiality of communications in the aggregate, you have to consider all users, both individually and in groups.  The multi-user setting considers the possibility that an attacker might want to attack any user from a group.  This is different than the single-user setting where we consider the advantage an attacker gains against a specific user because the attacker doesn't care which user gets hit.

Then you can say that an analysis based on a group that includes all users, recognizing that this is the pessimal case, might be performed to determine numbers.  However, the target for attacker advantage in that context might be different than targets in the single-user context.

Reply:
I think this is resolved now. @martinthomson, please have a look!